local/kind/setup.sh

```diff
@@ -95,11 +97,11 @@ for cluster in "${CLUSTERS[@]}"; do
     if [[ -z "${clusterExists}" ]]; then
         echo "Creating kind cluster ${cluster}"
         cp "${cluster}.config" "${TMP_DIR}/${cluster}.config"
-        KIND_EXPERIMENTAL_PROVIDER=${KIND_EXPERIMENTAL_PROVIDER:-} ${KIND_CMD} create cluster \
+        ${KIND_CMD} create cluster \
             --config "${TMP_DIR}/${cluster}.config" \
             --kubeconfig "${TMP_DIR}/${cluster}.kubeconfig"
 
-        if [[ ${KIND_CMD} == "sudo kind" ]]; then
+        if [[ ${KIND_CMD} == "sudo KIND_EXPERIMENTAL_PROVIDER=podman kind" ]]; then
```

Comment:
Would `[[ "${CONTAINER_ENGINE}" != "docker" ]]` or `[[ "${CONTAINER_ENGINE}" == "podman" ]]` achieve the same result? It might be easier to read.

Reply:
No it is not. Key here is that when sudo is used read rights need to be granted to the user.